src/actionId.ts

```diff
@@ -206,7 +220,8 @@ async function getActionIdSource(
     }
   } else {
     const adaptorTask = new Task('20220325-authorizer-adaptor', TaskMode.READ_ONLY, task.network);
-    return { useAdaptor: true, actionIdSource: await adaptorTask.deployedInstance('AuthorizerAdaptor') };
+    const actionIdSource = await adaptorTask.optionalDeployedInstance('AuthorizerAdaptor');
+    return { useAdaptor: true, actionIdSource };
```

Comment:
Should useAdaptor still be true when actionIdSource is undefined? If so, should document, since it's a bit confusing.

Reply:
yeah more than `useAdaptor` it is "doesn't have action ID identifier", which is the same as before in reality.